src/components/search-query.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import algoliasearch from 'algoliasearch';
+import PropTypes from 'prop-types';
+
+import { RefinementList, InstantSearch, SearchBox, Hits, ClearRefinements, Pagination } from 'react-instantsearch-dom';
+import webComponent from '../web-component';
+
+const searchClient = algoliasearch('H897LKXYG1', '50a4f124e0d934cac92e79ece376316a');
+
+const hitComponent = ({ hit }) => (
```

Comment:
Our React components should follow React naming conventions and start with a capital letter.
```suggestion
const HitComponent = ({ hit }) => (
```